src/utils.rs

```diff
@@ -178,32 +178,31 @@ pub mod test_utils {
     pub(crate) use references;
 
     macro_rules! vm_with_range_check {
-        () => {
-            VirtualMachine::new(
+        () => {{
+            let mut vm = VirtualMachine::new(
                 BigInt::new(Sign::Plus, vec![1, 0, 0, 0, 0, 0, 17, 134217728]),
-                vec![(
-                    "range_check".to_string(),
-                    Box::new(RangeCheckBuiltinRunner::new(bigint!(8), 8)),
-                )],
                 false,
-            )
-        };
+            );
+            vm.builtin_runners = vec![(
+                "range_check".to_string(),
+                Box::new(RangeCheckBuiltinRunner::new(bigint!(8), 8)),
+            )];
+            vm
+        }};
```

Comment:
This looks strange, why do we have extra braces for the block?

Reply:
The extra braces are needed because of the let statement